kano/network.py

```diff
@@ -384,6 +384,29 @@ def wpa_conf(essid, psk, confile, wep=False):
     Prepare and save a configuration file for WPA Supplicant daemon
     '''
 
+    #
+    # Support for Wireless channel 13, by using a country code.
+    # Force it by setting KANO_WIFI_COUNTRY envvar to the ISO/IEC alpha2
+    # country code format ("ES", "US", etc).
+    #
+    # Otherwise it is currently set automatically if ES locales are detected,
+    # in the form "es_AR.UTF-8" through the LANG envvar, translated to "ES".
+    #
+    # TODO: expect to have more mappings, i.e. en_US.UTF-8 => US
+    # Query current wireless country with "sudo iw reg get"
+    #
+    country_code=None
+    try:
+        cc=os.getenv('KANO_WIFI_COUNTRY')
+        if not cc:
+            cc=os.getenv('LANG')
+
+        cc = cc.split('_')[0]
```

Comment:
Do we really want the language part (es), not the country part (AR)? Does Argentina use the same wifi channels as Spain?